centreon/www/class/centreonStatistics.class.php

```diff
@@ -407,6 +407,48 @@ public function getAccData(): array
         return $data;
     }
 
+    /**
+     * Get poller/agent configuration data.
+     *
+     * @return array
+     */
+    public function getAgentConfigurationData(): array
+    {
+        $data = ['total' => 0];
+
+        $statement = $this->dbConfig->executeQuery(
+            <<<'SQL'
+                SELECT count(id)
+                FROM agent_configuration ac
+                SQL
+        );
+
+        $total = $statement->fetchColumn();
```

Comment:
Now, with CentreonDB, you must not to use PDO native, but the metods of CentreonDB ;) 
```suggestion
        $total = $this->dbConfig->fetchColumn($statement);
```

Reply:
ok